dotcom-rendering/playwright/tests/article.interactivity.e2e.spec.ts

```diff
@@ -227,7 +227,12 @@ test.describe('Interactivity', () => {
 				await loadPage({ page, path: `/Article/${articleUrl}` });
 
 				await page.locator('[data-testid=veggie-burger]').focus();
-				await page.locator('[data-testid=veggie-burger]').press('Tab');
+				// TODO tabbing from the burger element now takes two presses to get to the sub nav
+				// The first tab goes to the hidden expanded menu element
+				// Replicated in PROD. possible a11y regression
```

Comment:
Raised ticket for investigation here: https://github.com/guardian/dotcom-rendering/issues/14031